package.json

```diff
@@ -4,9 +4,13 @@
   "private": true,
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "format": "prettier --write \"**/*.+(ts|tsx|js|jsx|json|css)\"",
+    "build": "webpack",
+    "dev": "webpack serve"
```

Comment:
What does the `dev` command do? Does it start the dev mode with an automatic application update after saving? If true, what is the difference with the `start` command?